### DISCUSSION TITLE
[Feature] Import two-way of folder-albums

### DISCUSSION THREAD

**Discussion by picarica** [immich-app/immich#11935] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

could be dupe of #11668 but mine use case would be a little different or #5409 

only reason why i cant switch from nextcloud solution is that i like to use folder albums, its easy to keep it organized current workflow

1. take picture
2. open system gallery
3. find picture in camera folder, click > Move > Select desired album
4. FolderSync, syncs it to nextcloud instance, and is also viewable in cloud perfectly organized and will adjust when later on images are reorganized
5. easy done

impossible on immich, the feature would be to import and keep local albums to import into cloud albums, and then all images could be added to albums via immich app, so no usage of system gallery.

not sure if i am only one using albums this way, but it helped it keep it organized and find anything i wanted, i have split albums for my different pets, for members of family (this is solved by face recognition) then some scenery, important images, like some keys recovery for steam etc. and screenshot.

so some feature to easy to migrate, and keep it easy organized all the way, somehow to see on the app images that are in any albums now that album is called "Camera" so something equal in immich would be nice.

thank you

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678
